bears/rest/RSTcheckBear.py

```diff
@@ -32,16 +44,39 @@ class RSTcheckBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
-                         code_block_language_ignore: list=()):
+                         code_block_language_ignore: list=(),
+                         ignore_directives: list=(),
+                         ignore_roles: list=()):
         """
         :param code_block_language_ignore:
             Comma seperated list for languages of which code blocks
             should be ignored. Code block of following languages
             can be detected: ``bash``, ``c``, ``cpp``, ``json``,
             ``python``, ``rst``.
+
+        :param ignore_directives:
+             Comma sepereated list of directives to be ignored
+
+        :param ignore_roles:
+             Comma seperated list of roles to be ignored
         """
+
```

Comment:
but if you want to (which i don't recommend for consistency ) do it in another commit